default/generated/technology-usage/188-3rd-party.windup.yaml

```diff
@@ -339,3 +343,114 @@
     - java.dependency:
         lowerbound: 0.0.0
         name: com.alibaba.cloud.spring-cloud-circuitbreaker-sentinel
+- customVariables: []
+  description: Embedded framework - Rocket MQ
+  labels:
+  - konveyor.io/include=always
+  links: []
+  message: The application embeds the Rocket MQ framework.
+  ruleID: 3rd-party-30200
+  tag:
+  - Rocket MQ
+  - Embedded framework - Rocket MQ
+  when:
+    or:
+    - java.dependency:
+        lowerbound: 0.0.0
+        name: org.apache.rocketmq.rocketmq-client
+    - java.dependency:
+        lowerbound: 0.0.0
+        name: org.apache.rocketmq.rocketmq-acl
+    - java.dependency:
+        lowerbound: 0.0.0
+        name: com.alibaba.cloud.spring-cloud-starter-stream-rocketmq
+    - java.dependency:
+        lowerbound: 0.0.0
+        name: com.alibaba.cloud.spring-cloud-starter-bus-rocketmq
+- customVariables: []
+  description: Embedded framework - Cloud Seata
+  labels:
+  - konveyor.io/include=always
```

Comment:
Nit: Would it be better to indent the items a bit more? It may help for readability. E.g.,

```yaml
- customVariables: []
  description: Embedded framework - Cloud Seata
  labels:
    - konveyor.io/include=always
```

Reply:
This change is required across multiple files. It would be best to handle the refactoring in a separate PR dedicated solely to that task. I will open-up a follow-up PR. Thanks!